### ISSUE TITLE
Bug: `DateHelperTest::testNowSpecific` depends on time

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/CodeIgniter4#5492]
```
There was 1 failure:

1) CodeIgniter\Helpers\DateHelperTest::testNowSpecific
Failed asserting that 4 is equal to 1 or is less than 1.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Test/CIUnitTestCase.php:461
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Helpers/DateHelperTest.php:35
```
https://github.com/codeigniter4/CodeIgniter4/runs/4593459157?check_suite_focus=true

**Comment by kenjis** [comment]
```
1) CodeIgniter\Helpers\DateHelperTest::testNowSpecific
Failed asserting that 4 is equal to 1 or is less than 1.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Test/CIUnitTestCase.php:478
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Helpers/DateHelperTest.php:36
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:97
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/3317017866/jobs/5479434095

**Comment by kenjis** [comment]
Closed by #6753
